content/administration/maintain/enterprise.rst

```diff
@@ -1,9 +1,9 @@
 
 .. _setup/enterprise:
 
-===============================
-Upgrade Community to Enterprise
-===============================
+===================================
+Switch from Community to Enterprise
```

Comment:
Makes sense; especially since we reserve the term "upgrade" for a version change. Ok for you @xpl-odoo?

Reply:
This change was suggested by @xpl-odoo himself.